psalm.xml

```diff
@@ -5,14 +5,35 @@
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     name="Example Psalm config with recommended defaults"
     useDocblockTypes="true"
-    totallyTyped="false"
+    totallyTyped="true"
     allowPhpStormGenerics="true"
 >
     <projectFiles>
         <directory name="src"/>
     </projectFiles>
 
     <issueHandlers>
-        <PropertyNotSetInConstructor errorLevel="suppress"/>
+        <PropertyNotSetInConstructor>
+            <errorLevel type="suppress">
+                <file name="src/Git/CheckedOutRepository.php" />
+                <file name="src/Git/Revision.php" />
+                <file name="src/Changes.php" />
+            </errorLevel>
+        </PropertyNotSetInConstructor>
+
+        <MixedAssignment>
```

Comment:
Constants and default values can be a recursive structure `data ConstValue = int | int | string | float | bool | null | array ConstValue`, not really `mixed`

Reply:
Yeah but `mixed` basically means "I don't know what this is", appropriate for this use case. I guess I could add a `mixed-const` type, with rules around that, but `mixed` is far simpler. If I added `mixed-const` I'd also want to add `mixed-array-key` to represent array types (currently represented by `mixed` even though they can only be `string` or `int`).